src/Execution/Arguments/TypedArgs.php

```diff
@@ -181,6 +181,11 @@ protected function wrapWithType($valueOrValues, $type)
      */
     protected function wrapWithNamedType($value, NamedType $namedType)
     {
+        // As GraphQL does not allow empty input objects, we return null as is
+        if ($value === null) {
+            return $value;
```

Comment:
Not sure if it actually have a performance impact, but it should be easier for the code optimizer to optimize it when it's not a variable.

```suggestion
            return null;
```